recipes/rply/bld.bat

```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install 
```

Comment:
Could you remove the `bld.bat` and `build.sh` files in favor of something like:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26